doc/classes/@GlobalScope.xml

```diff
@@ -646,6 +646,16 @@
 				[b]Note:[/b] Consider using [method push_error] and [method push_warning] to print error and warning messages instead of [method print]. This distinguishes them from print messages used for debugging purposes, while also displaying a stack trace when an error or warning is printed.
 			</description>
 		</method>
+		<method name="print_rich" qualifiers="vararg">
+			<description>
+				Converts one or more arguments of any type to string in the best way possible and prints them to the console. Using BBCode for colors, italics and bold is possible.
+				[codeblock]
+				var a = [1, 2, 3]
+				print("a", "b", a) # Prints ab[1, 2, 3]
+				[/codeblock]
```

Comment:
This example uses `print` instead of `print_rich`. I would also add some BBCode to the arguments.

Reply:
Made a fitting example which uses some BBCode instead and also changed it to print_rich.

Reply:
You forgot the quotes for the string argument in the example.